aim/sdk/objects/image.py

```diff
@@ -222,6 +224,18 @@ def _from_tf_tensor(self, tensor, params):
             pil_image = PILImage.fromarray(array)
         self._from_pil_image(pil_image, params)
 
+    def _from_matplotlib_figure(self, figure, params):
+        try:
+            assert hasattr(figure, 'savefig')
+        except AssertionError:
+            raise ValueError('Cannot convert from matplotlib figure.')
+
+        buffer = BytesIO()
+        figure.savefig(buffer)
+        buffer.seek(0)
+
+        return self._from_pil_image(PILImage.open(buffer), params)
```

Comment:
It's OK for now to use `PILImage` as intermediate object to convert to png/jpeg. However it would be good to write `buffer` directly and make sure the appropriate metadata (`'source'`, `'format'`, etc.) is set correctly. Please open a Github issue for that.